RevPi-HAT-EEPROM-Format.md

```diff
@@ -1,145 +1,80 @@
+      - [Vendor info atom data (type=0x0001):](#vendor-info-atom-data-type0x0001)
+      - [GPIO map atom data (type=0x0002):](#gpio-map-atom-data-type0x0002)
+      - [Device Tree atom data (type=0x0003):](#device-tree-atom-data-type0x0003)
+
+## Revisions of this Document
+
+| Version | Date       | Changes |
+|---------|------------|---------|
+| 1.0     | 2022-06-10 | Initial release
+| 1.1     | 2022-09-06 | On strings no `\0` termination is needed. Restructure the document. Move the _Raspberry Pi HAT ID EEPROM FORMAT SPECIFICATION_ to the Appendix.
 
-### EEPROM Header Structure
-
-```text
-  Bytes   Field
-  4       signature   signature: 0x52, 0x2D, 0x50, 0x69 ("R-Pi" in ASCII)
-  1       version     EEPROM data format version (0x00 reserved, 0x01 = first version)
-  1       reserved    set to 0
-  2       numatoms    total atoms in EEPROM
-  4       eeplen      total length in bytes of all eeprom data (including this header)
-```
-
-### Atom Structure
-
-```text
-  Bytes   Field
-  2       type        atom type
-  2       count       incrementing atom count
-  4       dlen        length in bytes of data+CRC
-  N       data        N bytes, N = dlen-2
-  2       crc16       CRC-16 of entire atom (type, count, dlen, data)
-```
-
-### Atom Types
-
-```text
-  0x0000 = invalid
-  0x0001 = vendor info
-  0x0002 = GPIO map
-  0x0003 = Linux device tree blob
-  0x0004 = manufacturer custom data
-  0x0005-0xfffe = reserved for future use
-  0xffff = invalid
-```
+## RevPi Hat EEPROM Format Specification (v1)
 
-#### Vendor info atom data (type=0x0001):
+The RevPi HAT EEPROM format is based on the [Raspberry Pi HAT ID EEPROM FORMAT SPECIFICATION](https://github.com/raspberrypi/hats/blob/9616b5cd2bdf3e1d2d0330611387d639c1916100/eeprom-format.md) (RPi Hat Spec.). The data supplied this way is added to the device tree by the bootloader. A `hat` node is created below the root of the device tree. It can be accessed at runtime through the procfs: `/proc/device-tree/hat/`. The `hat` node can contain various attributes.
 
-Note that the UUID is mandatory and must be filled in correctly according to RFC 4122
-(every HAT can then be uniquely identified). It protects against the case where a user
-accidentally stacks 2 identical HATs on top of each other - this error case is only
-detectable if the EEPROM data in each is different. The UUID is also useful for
-manufacturers as a per-board 'serial number'.
-
-```text
-  Bytes   Field
-  16      uuid        UUID (unique for every single board ever made)
-  2       pid         product ID
-  2       pver        product version
-  1       vslen       vendor string length (bytes)
-  1       pslen       product string length (bytes)
-  X       vstr        ASCII vendor string e.g. "ACME Technology Company"
-  Y       pstr        ASCII product string e.g. "Special Sensor Board"
-```
+The mandatory atoms of the _Raspberry Pi HAT ID EEPROM FORMAT SPECIFICATION_ are also mandatory. For details see .
```

Comment:
maybe write -> are also mandatory for this format extension.

Is there anything missing after "For details see ." ?